tests/unit/model/test_element.py

```diff
@@ -19,6 +19,7 @@
 import pytest
 
 from structurizr.model.element import Element
+from structurizr.model.model import Model
```

Comment:
Using `Model` in these tests arguably makes them integration tests.

Reply:
Good point.  I'll extend the mock to include the `add_relationship` method I need instead.